jetty-ee10/jetty-ee10-annotations/src/main/java/org/eclipse/jetty/ee10/annotations/AnnotationParser.java

```diff
@@ -472,6 +472,12 @@ public void visit(final int version,
                           final String superName,
                           final String[] interfaces)
         {
+            //Check that the named class exists in the containingResource at the correct location.
+            //eg given the class with name "com.foo.Acme" and the containingResource "jar:file://some/place/something.jar!/"
+            //then the file "jar:file://some/place/something.jar!/com/foo/Acme.class" must exist.
+            if (_containingResource.resolve(name + ".class") == null)
```

Comment:
this is causing issues for osgi bundles as classes are not resolved anymore within bundles